// @salutejs/client/esm/index.js
export { createAssistant } from './createAssistant.js';
export { createAssistantHostMock, createAssistantHostMockWithRecord } from './mock.js';
export { e as createAnswerFromMockByMessageGetter, f as createMockRecorder, d as createMockWSCreator, b as createOnlineRecordPlayer, a as createRecordPlayer } from './record-30f9fe6e.js';
export { createAssistantDev, createSmartappDebugger } from './createAssistantDev.js';
import { b as asrExports } from './assistant-a6b726e2.js';
export { c as createAssistantClient, c as createAssistantSDK, a as createNavigatorAudioProvider } from './assistant-a6b726e2.js';
import { p as protoExports } from './sdk-f73acc07.js';
export { M as MessageNames, V as VpsVersion } from './sdk-f73acc07.js';
import './common-ba25e019.js';



var GetHistoryResponse = protoExports.GetHistoryResponse;
var InitialSettings = protoExports.InitialSettings;
var PacketWrapperFromServer = asrExports.PacketWrapperFromServer;
export { GetHistoryResponse, InitialSettings, PacketWrapperFromServer };
